technologies/app/rstudio-4.1/rstudio-4.1/Dockerfile

```diff
@@ -0,0 +1,101 @@
+FROM rocker/tidyverse:4.1.1
+
+LABEL maintainer="Saagie <tony@saagie.com>"
+
+USER root
+
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV CDH_MAIN_VERSION 5
+ENV CDH_VERSION ${CDH_MAIN_VERSION}.7.1
+ENV HADOOP_VERSION 2.7.0
```

Comment:
Hadoop 2.7 and on rstudio 3.6.2, it's hadoop 2.6?

Reply:
hadoop command line 2.7 adds recursive ls, it's nice but not mandatory, I can change it to 2.6 if necessary